build.gradle

```diff
@@ -19,6 +21,8 @@ repositories {
 	mavenCentral()
 }
 
+apply from: "gradle/spotless.gradle"
```

Comment:
"apply" is not necessary cf. https://plugins.gradle.org/plugin/com.diffplug.spotless